### DISCUSSION TITLE
Deploy as another user besides dokku ?

### DISCUSSION THREAD

**Discussion by cburkins** [dokku/dokku#4850] [Q&A] [answered]
I think I'm correct in saying that from the "client", there's essentially an ssh happening to the remote, and it's trying to log in as "dokku".    In my particular config, I can't allow "dokku" to have a ssh capability on that server.   Is it possible to have the remote setup as user@ip-addr instead of dokku@ip-addr ?   

Is that supported ?

**Comment by josegonzalez** [comment, marked as answer]
There isn't a way to do this with the same ssh key, but if the user has two ssh keys, one can be used for this purpose.

The underlying trigger here is sshcommand. The configured ssh key in Dokku triggers sshcommand - you can check `/home/dokku/.ssh/authorized_keys` to see how that works - with a `/home/dokku/.sshcommand` file as the proxy. If you setup something similar for the other users, you can trigger a dokku command (deploys or otherwise) using that second ssh key.

This was also answered in #4851.
